public/electron/main.js

```diff
@@ -141,6 +149,11 @@ function handleCheckSerialPort() {
   setUpPort().then(() => handleEventSend(TRIGGER_CODES.eventCodes.test_connect));
 }
 
+/**
+ * Sends the event_codes to the trigger port
+ * @param {} event The serial port event
+ * @param {number} code The event code to be recorder
```

Comment:
```suggestion
 * @param {number} code The event code to be recorded
```
I'm guessing the intended word is recorded?